app/views/layouts/_debug.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if Rails.application.secrets.display_debug.present? -%>
```

Comment:
Instead of putting it into secrets, would it be more prudent to put a value on the application instance? That way we could turn on debugging for individual instances?

Reply:
I patterned this after the way that [show_environment](https://github.com/atomicjolt/lti_starter_app/blob/sc/debug-info/app/views/layouts/_show_environment.erb) works. I figured that this would only really be useful in places where that would be as well. 

I figured it'd be useful in local development or in the dev environment. In those instances, I don't feel like there would be a need to have it toggled on an application basis. However, if it'd be useful on anything clients may see (beta or prod) then it would make more sense. Is that what  you're thinking?

Reply:
I figured that's why it was added was for debugging prod. But if this is more of a dev only thing, then in the secrets is fine with me.

Reply:
Do you think it would be useful for prod? If you do, then I do agree it would be nice to have it be on a application-by-application basis

Reply:
I can see it being useful. And you wouldn't need to add a database value. Just add it to the jsonb config.

Reply:
But if it isn't desired, then I'm good with these changes as is.